app/src/main/res/layout/activity_parking_reservation.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/main_background">
+
+    <TextView
+        android:id="@+id/text_dialog_fragment_parking_size"
+        style="@style/TextStyleParkingReservationActivityTitle"
+        android:text="@string/title_activity_parking_reservation"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <EditText
+        android:id="@+id/date_picker_input_parking_reserve_initial_date"
+        style="@style/EditTextStyleParkingReservationActivity"
+        android:autofillHints="none"
+        android:hint="@string/date_picker_input_parking_reservation_hint_for_initial_date"
+        android:inputType="datetime"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/text_dialog_fragment_parking_size" />
+
+    <EditText
+        android:id="@+id/date_picker_input_parking_reserve_end_date"
+        style="@style/EditTextStyleParkingReservationActivity"
+        android:autofillHints="none"
+        android:hint="@string/date_picker_input_parking_reservation_hint_for_end_date"
+        android:inputType="datetime"
```

Comment:
android:autofillHints & android:inputType could be into the style

Reply:
It is not working, it is still requested on layout